[glibmm]glibmmconfig.h missing

**Describe the bug**
glibmmconfig.h is missing.

**Environment**
- OS: Windows 10 20H2
- Compiler: Visual Studio 2019 16.10.3

**To Reproduce**

1. ./vcpkg install cairo cairomm pango pangomm

1. Set External Include Directory project setting to `$(VCPkgRoot)\installed\$(LibrariesArchitecture)-windows\include\sigc++-3.0;$(VCPkgRoot)\installed\$(LibrariesArchitecture)-windows\include\giomm-2.68;$(VCPkgRoot)\installed\$(LibrariesArchitecture)-windows\include\glibmm-2.68;$(VCPkgRoot)\installed\$(LibrariesArchitecture)-windows\include\cairomm-1.16;$(VCPkgRoot)\installed\$(LibrariesArchitecture)-windows\include\pangomm-2.48;$(ExternalIncludePath)`

1. Compile the code below.
`#include <glibmm.h>`

**Expected behavior**
No error.

**Failure logs** 
`C:\work\Project\vcpkg\installed\x64-windows\include\glibmm-2.68\glibmm.h(96,10): fatal error C1083: include ファイルを開けません。'glibmmconfig.h':No such file or directory`

**Additional context**
`#include <pangomm.h>` emits the same error.
See #18760 for the External Include Directory project setting above.

--- a/ports/pangomm/portfile.cmake
+++ b/ports/pangomm/portfile.cmake
@@ -1,9 +1,9 @@
 vcpkg_fail_port_install(ON_ARCH "arm" "arm64")
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://ftp.gnome.org/pub/GNOME/sources/pangomm/2.48/pangomm-2.48.0.tar.xz"
-    FILENAME "pangomm-2.48.0.tar.xz"
-    SHA512 0c1be5726740669c366214caf8b8b0bca1c7b223aafb2e6ce64f7a5a90d07c62ee509821ac787f3997ae78aa3a3ffd5cff7d33c73bc7ebd7fe642c56689d98a3
+    URLS "https://ftp.gnome.org/pub/GNOME/sources/pangomm/2.48/pangomm-2.48.1.tar.xz"
+    FILENAME "pangomm-2.48.1.tar.xz"
+    SHA512 c204a1cb7a404f055d62880a760716c5567a835ca495cc5e01589ed59fabb332490e529de716c3438cefbc4428c025d6d01c71e4412b2843e3ab3a3175ccc2f4
 )
 
 vcpkg_extract_source_archive_ex(

--- a/ports/pangomm/vcpkg.json
+++ b/ports/pangomm/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "pangomm",
-  "version": "2.48.0",
+  "version": "2.48.1",
   "description": "pangomm is the official C++ interface for the Pango font layout library. See, for instance, the Pango::Layout class.",
   "homepage": "https://ftp.gnome.org/pub/GNOME/sources/pangomm",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4953,7 +4953,7 @@
       "port-version": 1
     },
     "pangomm": {
-      "baseline": "2.48.0",
+      "baseline": "2.48.1",
       "port-version": 0
     },
     "parallel-hashmap": {

--- a/versions/p-/pangomm.json
+++ b/versions/p-/pangomm.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1ff0b71c742896ffb9da84638f7e20b2960aff80",
+      "version": "2.48.1",
+      "port-version": 0
+    },
+    {
       "git-tree": "d751b0dbf773ade4811729e3a55a8828dac8a78a",
       "version": "2.48.0",
       "port-version": 0

